packages/riipen-ui/src/components/Chip.test.jsx

```diff
@@ -5,13 +5,18 @@ import React from "react";
 import Chip from "./Chip";
 
 describe("<Chip>", () => {
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
```

Comment:
We will have to include this same block of code inside of every single test file we write, which isn't very DRY.

Perhaps there is a better way we can do this.

Take a look at this SoF question and the accepted answer: https://stackoverflow.com/questions/48722930/running-a-global-test-setup-before-each-test-in-jest

I think we should be able to add: 

```js
global.afterEach(() => {
   jest.restoreAllMocks();
});
```

inside of our `test/setup.js` file. That way we only have to write this code once.